### DISCUSSION TITLE
Are there any recipe?

### DISCUSSION THREAD

**Discussion by CodeKartonka** [DEM-Network-Corporation/Recycling#7] [Q&A] [answered]
have this mod one recipe? or why there no information abouth the recipes?

**Comment by DEMnetwork** [comment]
Okay
I'll Anware You

**Comment by DEMnetwork** [comment, marked as answer]
In The The Wiki I Show 5 Of 37 Recipes I've Added IN this Mod
You Can View The Wiki At Gitbook Or [Click Here](https://dem-network-corporation.gitbook.io/recycling-wiki/)

**Reply by DEMnetwork** [reply]
And I've added Part Of The Recipes At THe Github Wiki In The Page 2
 
DEMnetwork edited this page on May 25 · [3 revisions](https://github.com/DEM-Network-Corporation/Recycling/wiki/Page-2/_history)

**Reply by DEMnetwork** [reply]
>**edit**: Now Are 40
